modules/administration/pages/content-lifecycle.adoc

```diff
@@ -120,6 +120,16 @@ When you have created your project, defined environments, and attached sources a
 Building applies filters to the attached sources and clones them to the first environment in the project.
 
 
+[NOTE]
+====
+About patch sharing
+
+When you clone a vendor channel multiple times in your organization (e.g. you have 2 content projects with the same source vendor channels), {productname} does not create new patch clones for each cloned channel. Instead of that, a single patch clone is shared between your cloned channels.
+
+One important consequence is, that if the vendor modifies a patch (e.g. the patch gets retracted, or the patch packages change) and you build one content project, all channels sharing the cloned patch be synchronized with the original (even the channels in other environments of your content project, or other content project channels in your organization).
+
+You can turn the automatic patch synchronization feature off in your organization settings. You can synchronize the patch later manually using the Patch Sync function (menu:Software[Manage > Channels], click the channel you want to synchronize, and follow the menu:Repositories[Sync]). Note that even manual patch synchronization affects all channels sharing the patch.
+====
```

Comment:
```suggestion
You can clone vendor channels multiple times within your organization, which results in having two or more content projects with the same source vendor channels.
In this case, {productname} does not create new patch clones for each cloned channel. 
Instead, a single patch clone is shared between all of your cloned channels.
This can cause problems if a vendor modifies a patch; for example if the patch is retracted, or the packages within the patch are changed.
When you build one of the content projects, all the channels that share the cloned patch are synchronized with the original by default, even if the channels are in other environments of your content project, or other content project channels in your organization.
You can change this behavior by turning off automatic patch synchronization in your organization settings. 
To manually synchronize the patch later for all channels sharing the patch, navigate to menu:Software[Manage > Channels], and click the channel you want to synchronize.
You can monitor progress by navigating to menu:Repositories[Sync]. 
```

Reply:
I like the suggested change, but the link to `Sync` is wrong. Sorry, I've messed it up. Also, the `Sync` screen does not allow you to monitor the progress. It only allows you to perform the action. Instead of the last 2 lines,  I'd suggest something like that:

"To manually synchronize the patch later for all channels sharing the patch, navigate to menu:Software[Manage > Channels], click the channel you want to synchronize and navigate to the [guimenu]``Sync`` subtab."


Moreover, I'd like to explicitly mention the effect on all channels, as it was mentioned in my changes: "Note that even the manual patch synchronization affects all channels sharing the patch.". We can even reword it somehow, but I'd  like this information not to be omitted.

Reply:
(Forgot to add your handle, doing that now @Loquacity  :) )

Reply:
I've just accepted your suggestion and pushed the changes I mentioned in my latest comment.